scenarios/arcaflow/kill-pod/workflow.yaml

```diff
@@ -0,0 +1,63 @@
+input:
```

Comment:
The dir structure needs to be scenarios/arcaflow/kill-pods/workflow.yaml to match the location in the Kraken config